drivers/src/lms.rs

```diff
@@ -301,6 +306,66 @@ impl Lms {
         Ok(HashValue::from(digest))
     }
 
+    #[cfg(feature = "hw-latest")]
+    fn hash_chain<const N: usize>(
+        &self,
+        sha256_driver: &mut impl Sha256Alg,
+        wnt_prefix: &mut [u8; 55],
```

Comment:
There are still some hard-coded values. Can we ensure all of these are defined?

Reply:
done